.github/workflows/main.yml

```diff
@@ -16,32 +16,6 @@ on:
   pull_request:
 
 jobs:
-  check:
```

Comment:
Should we change line 13 to:
name: PR Check CI (build)
and rename main.yml to build.yml?

Reply:
replied in https://github.com/apache/incubator-nuttx/pull/666